pages/common/smartctl.md

```diff
@@ -0,0 +1,23 @@
+# smartctl
+
+> View a disk's [SMART](https://en.wikipedia.org/wiki/S.M.A.R.T.) data and other information.
+
+- View SMART health summary:
+
+`sudo smartctl --health {{/dev/<device>}}`
+
+- View device information:
+
+`sudo smartctl --info {{/dev/<device>}}`
+
+- Begin a (short) self-test:
```

Comment:
You can take short out of brackets and remove the tokens around "short" in the example. That way, the example directly matches the description.